### PR TITLE
Hackdna/postgres conf update

### PR DESCRIPTION
**Pull request by hackdna**
Remove PostgreSQL server from EC2 instance, make it easier to connect to PostgreSQL server running on Vagrant via SSH from the host, and streamline dependency handling for Python.

### CODE REVIEW

File: deployment/refinery-modules/refinery/manifests/postgresql.pp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  else {
+    class { '::postgresql::globals':
+      version  => '9.3',
+      encoding => 'UTF8',
+      locale   => 'en_US.utf8',
+    }
+    ->
+    class { '::postgresql::server':
+      # to make remote connections via SSH tunnel easier
+      ipv4acls => ["host all ${::app_user} 127.0.0.1/32 trust"],
```
Comment (drj11) [inline deployment/refinery-modules/refinery/manifests/postgresql.pp]: Technically, the 127 net (RFC 1918) is a /8 (try, for example, ping 127.5.5.5).

I don't think that's likely to be an issue.

File: deployment/refinery-modules/refinery/manifests/postgresql.pp [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class refinery::postgresql {
+
+  if $::deployment_platform == 'aws' {
```
Comment (drj11) [inline deployment/refinery-modules/refinery/manifests/postgresql.pp]: I had no idea you could do this with puppet.

Comment (hackdna) [inline deployment/refinery-modules/refinery/manifests/postgresql.pp]: This is just a global variable that we set, like `$::app_user` for example. The plan is to replace all the globals with class parameters to make classes self-contained.

File: deployment/refinery-modules/refinery/manifests/postgresql.pp (hunk 1 above)
Comment (hackdna) [inline deployment/refinery-modules/refinery/manifests/postgresql.pp]: Right but in this case we only care about 127.0.0.1 (localhost).

**Pull request merged**